Firmware/messages.cpp

```diff
@@ -169,6 +169,7 @@ extern const char MSG_PAUSED_THERMAL_ERROR[] PROGMEM_I1 = ISTR("PAUSED THERMAL E
 #ifdef TEMP_MODEL
 extern const char MSG_THERMAL_ANOMALY[] PROGMEM_I1 = ISTR("THERMAL ANOMALY");////MSG_THERMAL_ANOMALY c=20
 extern const char MSG_TM_NOT_CAL[] PROGMEM_I1 = ISTR("Temp model not calibrated yet.");////MSG_TM_NOT_CAL c=20 r=4
+extern const char MSG_TM_ACK_ERROR[] PROGMEM_I1 = ISTR("Clear TM error");////MSG_TM_ACK_ERROR c=18
```

Comment:
Adding new message needs an update of the pot and po files.
Also the new message should be translated.
Do want to do that in a separate PR?
Splitting "unblocks" this PR as we don't need to wait for the translations, which may take a while.

Reply:
Why would it block it?

Reply:
Because we have to wait for the translations